pyproject.toml

```diff
@@ -194,6 +192,32 @@ python_files = "test_*.py"
 
 norecursedirs = "__pycache__ build .pytest_cache *.egg-info .venv .github-venv"
 
+[tool.tox]
+legacy_tox_ini = """
+    [tox]
+    envlist = py310
+
+    [testenv]
+    # these options make Tox use the existing .venv dir created by venv
+    envdir = {env:VENV:.venv}
+    usedevelop = true
+    recreate = false
+    skip_install = true
```

Comment:
**question (non-blocking):** Just running through the logic here:

* [`envdir = true`](https://tox.wiki/en/latest/config.html#env_dir) - re-use the existing virtualenv, great
* [`recreate = false`](https://tox.wiki/en/latest/config.html#recreate) - don't unconditionally recreate the virtualenv (although false does mean "leave it up to tox")
* [`skip_install = true`](https://tox.wiki/en/latest/config.html#skip_install) - makes sense, the package is already installed in the existing virtualenv

The one that I don't quite understand is [`usedevelop = true`](https://tox.wiki/en/latest/config.html#usedevelop) - "Install the current package in editable mode." Does that do anything while `skip_install` is true?

Reply:
I wondered the same - when I had originally tested with and without, not using it caused Tox to install its own env anyway. However, now I'm not able to reproduce it, so ... let's see what happens when I push that change to GitHub.

Reply:
> although false does mean "leave it up to tox"

Bummer - it looks like you cannot disable recreation altogether. 
https://github.com/tox-dev/tox/issues/1372
https://github.com/tox-dev/tox/issues/1372

Reply:
Alas!